### DISCUSSION TITLE
Getting the most recent results

### DISCUSSION THREAD

**Discussion by luciapicos1** [jolpica/jolpica-f1#105] [Q&A]
Will there be a way to retrieve the latest race results without specifying the round? In Ergast, you could simply use `/recent` to get the most recent results. Is something similar possible here?

**Comment by theOehrly** [comment]
With Ergast you can use `/current` for the current season and `/last` for the latest round. 

For example:

http://ergast.com/api/f1/current/last/results

This also works with jolpica-f1:

https://api.jolpi.ca/ergast/f1/current/last/results/

Both return the race results for the last Grand Prix.

I'm not aware of any `/recent` parameter for Ergast.
